.gitignore

```diff
@@ -38,7 +38,8 @@ src/rust/grapl-web-ui/frontend/*
 nomad/local/mount-grapl-root-as-volume.nomad
 
 # Distribution / packaging
-build/
+/src/js/engagement_view/build/
+/src/js/engagement_view/source_map_explorer_result.html
```

Comment:
Might be better to put these into `src/js/engagement_view/.gitignore` for now, per the comment at the top of that file.